baekjoon/factorial.js

```diff
@@ -0,0 +1,11 @@
+let input = require('fs').readFileSync('/dev/stdin').toString();
+const value = Number(input);
+
+function factorial(value) {
+    if (value === 1) return value;
+    if (!value) return 1;
+    return value * factorial(value - 1);
+}
```

Comment:
만약 1! 이면 값이 1이기 때문에 굳이 1과 0일 때를 나눠서 value 값을 지정하기 보단 if (value <= 1)일때 1값을 return 하도록 하는게 더 효율성이 좋을거 같아요! 
추가로 삼항연산자를 이용하여 작성하면 한 줄로 작성이 됩니당
```suggestion
function factorial(value) {
    return value <= 1 ? 1 : value * factorial(value - 1);
}
```